lib/matplotlib/collections.py

```diff
@@ -2019,6 +2019,15 @@ def set_paths(self):
     def get_datalim(self, transData):
         return (self.get_transform() - transData).transform_bbox(self._bbox)
 
+    def get_coordinates(self):
+        """
+        Return the x and y vertices of the mesh.
+
+        The shape of the returned coordinates is
+        (*meshWidth* + 1, *meshHeight* + 1, 2).
```

Comment:
We're removing the names *meshWidth*/*meshHeight* from code and docs.

Some thing like this should do:

```suggestion
        Return the vertex positions of the mesh as (M, N, 2) array.
        
        M, N are the number of rows / columns in the mesh.
        The last dimension specifies the components (x, y).
```

Reply:
Sounds good. The only issue is that it may be confusing to use (M, N) here to refer to the vertices when other places use that notation to refer to the number of Polygons/Quads.

Reply:
Yeah, technically this might be `(M+1, N+1, 2)` array. Would that be better?

More generally, I'm switch back and forward between the +/-1 interpretations, i.e.  what is the canonical quantity for the number (M, N), the quadrilaterals or the vertices/coordinates?

Reply:
Most of the previous documentation has had (M, N) as the number of quadrilaterals in the mesh, and to me that makes intuitive sense since that is the quantity we are most interested in. I'd vote for changing this to (M+1, N+1, 2) and explaining that here, but I also don't have a real strong preference.

Reply:
I do - the data is MxN, the co-ordinates follow that.  You could argue the coordinates are more important if they are twisting out some complicated shape, but 99.9% of the time this is tracking out a regular grid.

Reply:
I changed the text to explicitly call out and distinguish the quads and vertices in the statement.

> Return the vertices of the mesh as an (M+1, N+1, 2) array.
> 
> M, N are the number of quadrilaterals in the rows / columns of the mesh, corresponding to (M+1, N+1) vertices. The last dimension specifies the components (x, y).